docs/running-jobs/slurm-commands.md

```diff
@@ -158,22 +158,22 @@ $ sacct –-jobs=your_job-id –-starttime=YYYY-MM-DD --long
 Like `sstat`, the standard output of sacct may not provide the
 information we want. To remedy this, we can use the `--format` flag to
 choose what we want in our output. Similarly, the format flag is
-handled by a list of comma separated variables which specify output
+handled by a list of comma-separated variables that specify output
 data:
 
 ```bash
 $ sacct --user=your_rc-username --format=var_1,var_2, ... ,var_N
 ```
 
-A chart of some variables is provided below:
+A chart of some of these variables is provided below:
 
 Variable    | Description
 ------------|------------
 account     | Account the job ran under.
-avecpu      | Average CPU time of all tasks in job.
+avecpu      | Average CPU time of all tasks in the job.
 averss      | Average resident set size of all tasks in the job.
 cputime     | Formatted (Elapsed time * CPU) count used by a job or step.
-elapsed     | Jobs elapsed time formated as DD-HH:MM:SS.
+elapsed     | Jobs elapsed time formatted as DD-HH:MM:SS.
 exitcode    | The exit code returned by the job script or salloc.
 jobid       | The id of the Job.
 jobname     | The name of the Job.
```

Comment:
Could we replace the dates in this section with newer ones, like March and February of this year? 

![Screenshot 2024-09-19 at 4 22 13 PM](https://github.com/user-attachments/assets/1e21a44e-e756-4e55-8537-e183b8f4cdee)